src/data.rs

```diff
@@ -105,6 +110,37 @@ mod array_impls {
         })))
     }
 
+    pub(crate) fn create_boolean_array_with_errors<DT: CellType + DataType + Debug>(
+        data: &Range<DT>,
+        col: usize,
+        offset: usize,
+        limit: usize,
+    ) -> (Arc<dyn Array>, CellErrors) {
+        let mut cell_errors = CellErrors { errors: vec![] };
+
+        let arr = Arc::new(BooleanArray::from_iter((offset..limit).map(|row| {
+            data.get((row, col)).and_then(|cell| {
+                if cell.is_empty() {
+                    None
+                } else if let Some(b) = cell.get_bool() {
+                    Some(b)
+                } else if let Some(i) = cell.get_int() {
+                    Some(i != 0)
+                } else if let Some(f) = cell.get_float() {
+                    Some(f != 0.0)
+                } else {
+                    cell_errors.errors.push(CellError {
+                        position: (row - offset, col),
```

Comment:
Hmm I'm not sure the offset should be subtracted here: We still want the position of the cell in the Sheet, don't we ?

Maybe we could store the offset in the cell error as well, and provide another property to access the cell without the property if you need that ?

Reply:
I only need the position as it will correspond to a dataframe, but I've updated it so there is now `position` and `offset_position` to provide either